Controller/Standard/AbstractStandard.php

```diff
@@ -297,8 +300,8 @@ protected function _setQuote($quote)
    */
   protected function _getDbQuote($zip_checkout_id)
```

Comment:
Could we remove this function as we change to call API to get the previous checkout